src/components/TaskList.js

```diff
@@ -1,38 +1,45 @@
-import React from "react";
+import React from 'react';
 
-import Task from "./Task";
-import { isTag } from "postcss-selector-parser";
+import Task from './Task';
 
 const TaskList = props => {
-  console.log("ii", props.activeState);
+  console.log(props.todos);
+
+  let filteredContent = props.todos.filter(todo => {
+    return todo.value.indexOf(props.searchField) !== -1;
+  });
 
   let list = null;
-  if (props.activeState == "Completed") {
-    list = props.todos.filter(todo => todo.done);
-  } else if (props.activeState == "Incomplete") {
-    list = props.todos.filter(todo => !todo.done);
+  if (props.activeState == 'Completed') {
+    list = filteredContent.filter(todo => todo.done);
+  } else if (props.activeState == 'Incomplete') {
+    list = filteredContent.filter(todo => !todo.done);
+  } else {
+    list = [...filteredContent];
+  }
+  if (list.length != 0) {
+    return (
+      <div>
+        {list.map(todo => {
+          return (
+            <Task
+              key={todo.id}
+              index={todo.id}
```

Comment:
`id` should be better than `index`
```suggestion
              id={todo.id}
```

Reply:
You may not need this if you are passing `todo` as props.